packages/libs/pactjs-utils/tsconfig.json

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "./node_modules/@kadena/heft-rig/profiles/default/tsconfig-base.json",
+  "compilerOptions": {
+    "types": ["heft-jest", "node"]
```

Comment:
Does it work without "node"? We found out that adding this to the types, the libs potentially don't work when using in the browser.